DEBIAN/copyright

```diff
@@ -0,0 +1,23 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: gcsfuse
+Upstream-Contact: gcs-fuse-maintainers@google.com
+
+Files: *
+Copyright: Copyright 2020 Google Inc.
+License: Apache-2.0
+
+License: Apache-2.0
```

Comment:
This line is duplicate. Is it supposed to be this way?

Reply:
Yes this is in accordance to https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#stand-alone-license-stanza

Reply:
ok.